handlers/api/tagsApiHandler.js

```diff
@@ -0,0 +1,92 @@
+var tag = require('../../dao/tag.js');
+
+var sanitizer = require('sanitize-html');
+var apiutils = require('./apiUtils.js');
+
+var TagsApiHandler = function () {
+};
+
+TagsApiHandler.getTags = function (req, res) {
+    tag.getAll(function (result) {
+        res.writeHead(200, { "Content-Type": "application/json" });
+        res.end(JSON.stringify(result));
+    });
+};
+
+TagsApiHandler.getForProject = function (req, res) {
+    var projectId = sanitizer(req.params.projectId);
+    tag.getForProject(projectId, function (result) {
+        res.writeHead(200, { "Content-Type": "application/json" });
+        res.end(JSON.stringify(result));
+    });
+};
+
+TagsApiHandler.addTag = function(app) {
+    return function (req, res) {
+        req.checkBody('name', "Name cannot be blank.").notEmpty();
+
+        var errors = req.validationErrors();
+        if (errors) {
+            res.end(JSON.stringify({success: false, error: errors}));
```

Comment:
is there some kind of default head written? Don't see res.writeHead here (and a few other instances in this file)